src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -489,7 +489,6 @@ private static Document GetDocumentToVerify(DocumentId expectedChangedDocumentId
             int index, ImmutableArray<CodeAction> actions, CodeActionPriority? priority = null)
         {
             Assert.NotNull(actions);
-            Assert.NotEmpty(actions);
```

Comment:
What is this change and how is it related to Quick Info work?

Reply:
reverted. the change was accidentally got in when I was switching branches...